bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -50,13 +50,21 @@
 		<item-type>String</item-type>
 		<label>@text/channel-type.mielecloud.spinning_speed.label</label>
 		<description>@text/channel-type.mielecloud.spinning_speed.description</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Speed</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="spinning_speed_raw">
 		<item-type>Number</item-type>
 		<label>@text/channel-type.mielecloud.spinning_speed_raw.label</label>
 		<description>@text/channel-type.mielecloud.spinning_speed_raw.description</description>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
This should be the same as for the `spinning_speed` channel type, since the only difference is the representation (string or number). `Status` seems most correct, since the appliance wil not measure it, but just report it for the program.